PraceDomowe/PracaDomowa5/Nahorny_Jakub/Praca domowa 5.ipynb

```diff
@@ -0,0 +1,1763 @@
+{
```

Comment:
<p>To nie jest dobrze, przede wszystkim powinieneś użyć <code>.sum()</code> a nie <code>.count()</code>, a po drugie chodziło o wybranie z wyniku odpowiedniego rekordu z tabeli tak aby się wyświetlił jako wynik wykonania komórki, nie o odczytanie go z tabeli.</p><p> </p><p><code>0.2 punktu</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-ExploratoryDataAnalysis/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>